tests/script-based-pre/build-cache-dirty/rebuild.expected

```diff
@@ -0,0 +1,22 @@
+Initial compilation
+target/initial.log:5
+target/initial.log:Checking harness check_u8_i16...
+target/initial.log:Checking harness check_u8_u32...
+target/initial.log:Complete - 2 successfully verified harnesses, 0 failures, 2 total.
+Run with a new argument that affects compilation
+target/enable_checks.log:4
```

Comment:
Why did the number go down by 1 from `initial.log`?

Reply:
Sorry, I updated the test to use a crate with less dependencies. But that was a great question, so I reverted the test locally to double check. The reason is that one of the crates (`autocfg`) is a build-dependency. Here is the result of `cargo tree` for the original test:
```target_lib v0.1.0 (/kani/tests/script-based-pre/build-cache-dirty/target_lib)
└── num-bigint v0.4.3
    ├── num-integer v0.1.45
    │   └── num-traits v0.2.15
    │       [build-dependencies]
    │       └── autocfg v1.1.0
    │   [build-dependencies]
    │   └── autocfg v1.1.0
    └── num-traits v0.2.15 (*)
    [build-dependencies]
    └── autocfg v1.1.0
```

The cache for build dependencies doesn't get affected by the argument change, since the build dependency configuration uses host configuration.

Reply:
And here is the full compilation log:
```
Run with a new argument that affects compilation
   Compiling num-traits v0.2.15
   Compiling num-integer v0.1.45
   Compiling num-bigint v0.4.3
   Compiling target_lib v0.1.0 (/kani/tests/script-based-pre/build-cache-dirty/target/target_lib)
```

Reply:
Got it, thanks for clarifying.